transports/azure-service-bus/legacy/legacy-asb-warning.include.md

```diff
@@ -1 +1 @@
-WARNING: This page refers to the **deprecated** legacy Azure Service Bus transport that uses the [WindowsAzure.ServiceBus NuGet package](https://www.nuget.org/packages/WindowsAzure.ServiceBus/). All users should migrate to the [Azure Service Bus transport](/transports/azure-service-bus) transport.
+WARNING: This page refers to the legacy Azure Service Bus transport that uses the [WindowsAzure.ServiceBus NuGet package](https://www.nuget.org/packages/WindowsAzure.ServiceBus/). This package is currently on extended support until May, 2022 after which it will be out of support according to the [support policy](/nservicebus/upgrades/supported-versions.md#transport-packages-nservicebus-azure-transports-windowsazureservicebus). All users should migrate to the [Azure Service Bus transport](/transports/azure-service-bus) transport. 
```

Comment:
```suggestion
WARNING: This page refers to the legacy Azure Service Bus transport that uses the [WindowsAzure.ServiceBus NuGet package](https://www.nuget.org/packages/WindowsAzure.ServiceBus/). This package is currently on extended support until May 2022, after which it will be out of support according to the [support policy](/nservicebus/upgrades/supported-versions.md#transport-packages-nservicebus-azure-transports-windowsazureservicebus). All users should migrate to the [Azure Service Bus transport](/transports/azure-service-bus) transport. 
```